src/main/java/com/codegym/jrugotom5/service/AdvertService.java

```diff
@@ -71,7 +71,7 @@ public List<AdvertBasicInfoDTO> getByCategory(String category) {
     public void deleteAdvertById(Long id) {
         if (!advertRepository.existsById(id)) {
             log.error("Advert with ID {} not found", id);
-            throw new EntityNotFoundException("Advert with %d ID not found".formatted(id));
+            throw new AdvertDeletionException("Advert with %d ID not found".formatted(id));
```

Comment:
IMHO, It's not clear what does this exception mean.
I think the exception should be about the question of existence (as we throw it when we don't find any entities)
So I would use something like "EntityNotFoundException" (such an exception already exists in other PRs - check them)

BUT it's only my opinion. You can leave it out if the mentor doesn't mind

Reply:
done